modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -353,6 +370,14 @@ public void testGetObjectEntries() throws Exception {
 
 		// Equals/not equals expression
 
+		testGetObjectEntries(
+			HashMapBuilder.put(
+				"filter",
+				filterString.concat(
+					buildEqualsExpressionFilterString("flagged", true))
+			).build(),
+			objectEntry2, objectEntry4);
```

Comment:
Hey @jairmedeiros this use case should come after [this one](https://github.com/liferay-objects/liferay-portal/pull/3174/files#diff-82f88640eff05dedc724f92e76be2cf21cd6e4ffae22f9cba25e62e361ced5d0R406-R412), sort this as well,

Thank you!